core/data/src/main/java/com/yapp/core/data/remote/di/ApiModule.kt

```diff
@@ -45,4 +48,14 @@ internal object ApiModule {
     fun providePostsApi(retrofit: Retrofit): PostsApi {
         return retrofit.create(PostsApi::class.java)
     }
+
+    @Singleton
+    @Provides
+    fun provideAttendanceApi(retrofit: Retrofit): AttendanceApi {
+        return retrofit.create()
+    }
+
+    @Singleton
+    @Provides
+    fun providesSessionsApi(retrofit: Retrofit) = retrofit.create<SessionApi>()
 }
```

Comment:
```suggestion

    @Singleton
    @Provides
    fun provideAttendanceApi( @AuthRetrofitretrofit: Retrofit): AttendanceApi {
        return retrofit.create(AttendanceApi::class.java)
    }

    @Singleton
    @Provides
    fun providesSessionsApi(@AuthRetrofit retrofit: Retrofit): Session {
        return retrofit.create(SessionApi::class.java)
    } 
}
```

R: @AuthRetrofit 어노테이션을 붙여야 `AuthInterceptor` 가 붙은 레트로핏 클라이언트를 사용할 수 있어요.

Reply:
[fa9c1f5](https://github.com/YAPP-admin/yappu-world-android/pull/133/commits/fa9c1f55b5ca33c9c54517bd28f30e421f23aee8)

코멘트 적용 완료 처리했습니다 ~